lexik/jwt-authentication-bundle/2.3/post-install.txt

```diff
@@ -0,0 +1,7 @@
+<bg=blue;fg=white>              </>
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
```

Comment:
I would use a on-liner here instead of a block (to avoid having several such blocks when installing several packages with post-install output). What about something along the lines of:

```
<bg=blue;fg=white> Next  </>Generate a key pair for LexikJWTAuthenticationBundle
```